models/repo_collaboration.go

```diff
@@ -260,6 +259,20 @@ func reconsiderWatches(ctx context.Context, repo *repo_model.Repository, uid int
 	return removeIssueWatchersByRepoID(db.GetEngine(ctx), uid, repo.ID)
 }
 
+func getRepoTeams(e db.Engine, repo *repo_model.Repository) (teams []*Team, err error) {
```

Comment:
unrelated ?!?!

Reply:
I guess, you have an outdated main aka. master

Reply:
either "allow maintainer to edit" or add upstream remote and fetch changes :)

Reply:
its on top of `main` branch

Reply:
@flozzone `This branch is 2221 commits behind mgjm/gitea:master. ` ...

Reply:
edits by maintainers are allowed

Reply:
did you forked from another fork?

Reply:
tldr: you dont have to worry, just add upstream remote and use that

Reply:
why mgjm, need to counter check now

Reply:
i will squash my commits

Reply:
```sh
git add remote upstream https://github.com/go-gitea/gitea
git fetch upstream
git checkout -f upstream/main -- models/repo_collaboration.go
...
```

Reply:
I was using your checkout CLI line, whose upstream didn't match with mine. what a wonder :)